README.md

```diff
@@ -144,6 +144,7 @@ The Hardhat Chainlink plugin supports the following Chainlink services:
 - `automationRegistry` (Automation Registries)
 - `automationRegistrar` (Automation Registrars)
 - `vrf` (Verifiable Random Functions)
+- `functionsRouter` (Functions Service)
```

Comment:
the Router is a CL contract.  The services is just `Functions`.  We should rename this through the document and the code base.  Functions Router is a specific contract that CL controls and so we should not overload that terms -- it is not the "service" itself.

Reply:
Yes, you're absolutely right on it. The naming here not so clear, I need to re-do it for more clarity and transparency, it is related also to automations. Although at the moment the strict separation between wrappers for the Functions Router smart contract and other "service" methods is not entirely obvious and it’s worth working on it too.